src/InsetOrderOptimizer.cpp

```diff
@@ -216,6 +218,25 @@ BinJunctions InsetOrderOptimizer::variableWidthPathToBinJunctions(const Variable
             }
         }
     }
+
+    if (alternate_inset_direction) {
+        bool alternate_inset_print_direction = reverse_initial_inset;
+        for (PathJunctions &paths: insets) {
+            for (LineJunctions &line: paths) {
+                if (alternate_inset_print_direction) {
```

Comment:
code-style brackets on newline
```suggestion
        for (PathJunctions &paths: insets)
        {
            for (LineJunctions &line: paths)
            {
                if (alternate_inset_print_direction)
                {
```